arduino-ide-extension/src/browser/contributions/survey-notification.ts

```diff
@@ -6,18 +6,13 @@ import { nls } from '@theia/core/lib/common';
 import { WindowService } from '@theia/core/lib/browser/window/window-service';
 import { ArduinoPreferences } from '../arduino-preferences';
 
-export type Survey = {
-  url: URL;
-  id: string;
-};
-
 const SURVEY_MESSAGE = nls.localize(
   'arduino/survey/surveyMessage',
   'Please help us improve by answering this super short survey. We value our community and would like to get to know our supporters a little better.'
 );
 const DO_NOT_SHOW_AGAIN = nls.localize(
   'arduino/survey/dismissSurvey',
-  'DON’T SHOW ANYMORE'
+  "DON'T SHOW ANYMORE"
```

Comment:
```suggestion
  "DON'T SHOW AGAIN"
```

Reply:
Thanks for the review. I fixed this [here](https://github.com/arduino/arduino-ide/pull/1035/commits/ae828a5c102daedaa5ae31c58b76b1ba5fb04167).